pkg/gardenlet/controller/federatedseed/network-policy/controller.go

```diff
@@ -0,0 +1,202 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package network_policy
```

Comment:
```suggestion
package networkpolicy
```

Ref https://blog.golang.org/package-names:
> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.

(I'd also rename the directory from `network-policy` to `networkpolicy`)